akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -505,7 +514,7 @@ private[stream] final class GraphInterpreterShell(
       _isTerminated = true
```

Comment:
Ok, I add a comment for clarity. I won't bite today for "don't you agree" and "highly advisable", but please omit them next time, they are unnecessarily condescending. It is completely fine to just ask me to put in an additional comment.